pkg/catalog/outbound_traffic_policies.go

```diff
@@ -351,3 +381,46 @@ func (mc *MeshCatalog) ListMeshServicesForIdentity(identity identity.ServiceIden
 
 	return dstServices
 }
+
+func getRetryPolicy(t *access.TrafficTarget) (trafficpolicy.RetryPolicy, bool) {
```

Comment:
Instead of returning a bool here, maybe change return a pointer to the `RetryPolicy` struct so you can return `nil` in cases where the policy is not valid and the caller should ignore it.